.Documentation/articles/setup-guides/03-simulator.md

```diff
@@ -0,0 +1,76 @@
+# XRI Simulator
+
+The Unity's XRI Simulator comes in handy when you want to test your VR Application without headset.
+The XRI Simulator requires to use of the latest XR Interaction Toolkit version along with the new input system. Currently, the Creator is not compatible.
+Innoactive Creator automatically downloads and enables the XR Interaction Toolkit.
+
+
+## Step 1: Configure the use of the 'new input system' for your Project
+The XRI Simulator uses the Unity Input System to drive VR interactions. Make sure that the new input system is configured in your project.
+
+> Open _Edit > Project Settings_ 
+
+> Select the _Player_ tab in the opening window. 
+
+> Scroll down to _Active Input Handling_ and set it to either `both` or `new input system`.
+
+
+## Step 2: Check if you have required packages
+The Simulator uses a default set of input actions and presets used with the XR Interaction Toolkit behaviors that use the new Input System. It also delivers Assets related to the simulation of XR HMD and controllers. Both needs to be imported.
+
+> Open the Package Manager at Window > Package Manager 
+
+> Select 'in Project' in the drop-down on the top. 
+
+> Select the XR Interaction Toolkit and scroll the tab window down. Make sure that both `Default Input Actions` and `XR Device Simulator` are imported. 
+
+> Check also if the folder _Assets/Samples_ exists in the Unity Project tab.
+
+## Step 3: Change from XR Rig to XR Rig action-based
+
+> Select 'Setup Training Scene' from the Innoactive Menu drop-down.
+
+If your scene has the obsolete `[XR Setup]` object in the scene, a pop-up window allows you to delete it. You will not need it anymore.
+
+![Delete Obsolete Rig](../images/xr/dialogXRRigDeletion.png "A pop-up allows you to delete the obsolete `[XR Setup]` object from the scene").
+
+> delete the `[XR Setup]` object if it is in your object hierachy.
+
+Check the object hierachy tab in unity. You should see an `[INTERACTION_RIG_LOADER]` and a `[TRAINEE]` object.
+
+## Step 4: check the available Rigs
+>Select the `[INTERACTION_RIG_LOADER]`.
+
+Check the Unity Inspector. There should be an 'Interaction Rig Setup' script, which contains a list with available interaction Rigs. 
+- XR Rig Legacy for the legacy input
+- XR Rig for the new input system
+- none in case you want to use your custom XR Rig already on the scene.
```

Comment:
```suggestion
- `XR Rig` for the new input system
- `XR Rig Legacy` for the legacy input system
- `<None>` in case you want to use your custom XR Rig already in the scene.
```